packages/components/src/components/input-date/input-date.e2e.ts

```diff
@@ -60,6 +61,158 @@ test.describe('kol-input-date', () => {
 		});
 	});
 
+	test.describe('Consistent Return Value for type datetime-local', () => {
+		const testValues = [
+			{ label: 'ISO String', value: '2020-03-03' as Iso8601 },
+			{ label: 'Date Object', value: new Date('2020-03-03T03:02:01.099Z') },
+		];
+
+		testValues.forEach(({ label, value }) => {
+			test(`should dispatch onchange  with value as ${label} in event detail`, async ({ page }) => {
```

Comment:
Die Beschreibungen sind hier noch ein bisschen durcheinander, wir müssen zwischen den DOM-Events (z.B. "CustomEvent vom Type `kol-change`) und den Callbacks (z.B. `._on.onChange()`) richtig unterscheiden.

Ich würde die Beschreibungen wie folgt anpassen:

```diff
-should dispatch onchange  with value as ${label} in event detail
+should call onChange with value parameter as ${label}

-should dispatch onInput with value as ${label} in event detail
+should call onInput with value parameter as ${label}

-should return ${label} when initial value is a ${label} and trigger custom "change" DOM event
+should trigger custom "kol-change" DOM event with ${label} as event detail 

-should return ${label} when initial value is a ${label} and trigger custom "input" DOM event
+should trigger custom "kol-input" DOM event with ${label} as event detail

-should return the correct value with getValue() as ${label}
+should return the correct value for getValue() as ${label}
```

Eigentlich müssten wir überall noch am Ende schreiben `, when initial value is a ${label}`. Um das nicht so oft wiederholen zu müssen, würde ich aber lieber weitere Describe-Blöcke einführen.

Der Aufbau könnte dann sein:

```
Events
    when initial value is a ${label}
        should call onChange with value parameter as ${label}
        should call onInput with value parameter as ${label}
        should trigger custom "kol-change" DOM event with ${label} as event detail 
        should trigger custom "kol-input" DOM event with ${label} as event detail
        should return the correct value for getValue() as ${label}
```

Bzw. mit aufgelösten Variablen:

```   
Events
    when initial value is a ISO String
        should call onChange with value parameter as ISO String
        should call onInput with value parameter as ISO String
        should trigger custom "kol-change" DOM event with ISO String as event detail 
        should trigger custom "kol-input" DOM event with ISO String as event detail
        should return the correct value for getValue() as ISO String
    when initial value is a Date Object
        should call onChange with value parameter as Date Object
        should call onInput with value parameter as Date Object
        should trigger custom "kol-change" DOM event with Date Object as event detail 
        should trigger custom "kol-input" DOM event with Date Object as event detail
        should return the correct value for getValue() as Date Object
```